doc/getting_started/build_sw.md

```diff
@@ -223,12 +223,20 @@ Note, the file will **not** have an extension.
 A disassembly of all executable sections is produced by the build system by default.
 It can be found by looking for files with the `.dis` extension next to the corresponding ELF file.
 
+```console
+./bazelisk.sh build //sw/device/tests:uart_smoketest_prog_sim_verilator_dis
+
+less "$(./bazelisk.sh outquery //sw/device/tests:uart_smoketest_prog_sim_verilator_dis)"
```

Comment:
I figured this deserved an example, too :)

Reply:
good thinking!